### PR TITLE
Fix #142917 yay.space

### LINKED ISSUE #142917: yay.space
### Issue URL (Annoyance)
 [https://yay.space/group/268660](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fyay.space%2Fgroup%2F268660)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v4.1.53
Browser: | Chrome
Stealth mode: | disabled
Filters: | <b>Ad Blocking:</b><br/>AdGuard Base<br/><br/><b>Privacy:</b><br/>AdGuard URL Tracking, <br/>AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/>AdGuard Social Media<br/><br/><b>Annoyances:</b><br/>AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/>AdGuard Russian, <br/>AdGuard Chinese, <br/>AdGuard Japanese

### PR DESCRIPTION
**Pull request by SKEIDs**
### To avoid invalid pull requests, please check and confirm following terms

- [x] This is not an ad/bug report;
- [x] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [x] I have performed a self-review of my own changes;
- [x] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [ ] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [ ] Social media buttons — share, like, tweet, etc;
- [x] Annoyances — pop-ups, cookie warnings, etc;
- [ ] Filters maintenance.

### Enter the issue address

 Fix #142917 yay.space

### Add your comment and screenshots

1. `##.AppDownloadAds`
 	https://yay.space/group/268660

2. `##div[class^="Home__app-download-"]`
	https://yay.space/

3. `##.AnonymousCallListPopup`
	https://yay.space/

<details><summary>screenshot - 1</summary>

</details><br/>

<details><summary>screenshot - 2,3</summary>

</details><br/>

### Terms

- [x] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

File: AnnoyancesFilter/Popups/sections/popups_specific.txt [hunk 1]
```diff
@@ -12,6 +12,7 @@ forum.duolingo.com#?#.ReactModal__Overlay:has(> div.ReactModal__Content h2[class
 goo-net.com###floatingNaviCarlease
 aspor.com.tr###contactLink
 cnet.com###video-rectangle
+yay.space##.AnonymousCallListPopup
```
Comment (Alex-302) [inline AnnoyancesFilter/Popups/sections/popups_specific.txt:15]: Not annoying.

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/Popups/sections/popups_specific.txt (hunk 1 above)
Comment (Yuki2718) [reply AnnoyancesFilter/Popups/sections/popups_specific.txt:15]: IMO `##div[class^="Home__app-download-"]` too.

Comment (Alex-302) [reply AnnoyancesFilter/Popups/sections/popups_specific.txt:15]: @SKEIDs Please leave `yay.space##.AppDownloadAds` only.

Comment (SKEIDs) [reply AnnoyancesFilter/Popups/sections/popups_specific.txt:15]: I see

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**